src/main/java/past/cwdt/bookstore/model/Book.java

```diff
@@ -8,10 +8,14 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Table(name = "books")
 @Data
```

Comment:
@Data is not good, better add @Getter and @Setter

Reply:
Agree